lnrpc/routerrpc/router_backend.go

```diff
@@ -231,13 +231,41 @@ func (r *RouterBackend) QueryRoutes(ctx context.Context,
 		return nil, err
 	}
 
+	// Calculate route success probability. Do not rely on a probability
+	// that could have been returned from path finding, because mission
+	// control may have been disabled in the provided ProbabilitySource.
+	successProb := r.getSuccessProbability(route)
```

Comment:
This still seems to be sourcing from mission control though?

Reply:
Yes that is the intention. It could be that we passed a skewed probability function to path finding to control the resulting route, but we still want to know the actual mission control-based probability of that route. 

I was in doubt about that, but otherwise you'd always get probability `1` for queryroutes without `use_mc`. That is no info, so I figured it would be better to return something that can be useful in that case.